Pcap++/src/DpdkDeviceList.cpp

```diff
@@ -251,15 +251,26 @@ SystemCore DpdkDeviceList::getDpdkMasterCore()
 
 void DpdkDeviceList::setDpdkLogLevel(LoggerPP::LogLevel logLevel)
 {
+#if (RTE_VER_YEAR >= 16) && (RTE_VER_MONTH > 07)
+	if (logLevel == LoggerPP::Normal)
+		rte_log_set_global_level(RTE_LOG_NOTICE);
+	else // logLevel == LoggerPP::Debug
+		rte_log_set_global_level(RTE_LOG_DEBUG);
+#else
 	if (logLevel == LoggerPP::Normal)
 		rte_set_log_level(RTE_LOG_NOTICE);
 	else // logLevel == LoggerPP::Debug
 		rte_set_log_level(RTE_LOG_DEBUG);
+#endif
 }
 
 LoggerPP::LogLevel DpdkDeviceList::getDpdkLogLevel()
 {
+#if (RTE_VER_YEAR >= 16) && (RTE_VER_MONTH > 07)

```

Comment:
Same comment regarding the `#if`, in order for it to work on all DPDK version later than 17.11 you should change it to:
`#if (RTE_VER_YEAR > 17) || (RTE_VER_YEAR == 17 && RTE_VER_MONTH >= 11)`